src/cmd-buildextend-aws

```diff
@@ -7,8 +7,9 @@
 import os,sys,json,yaml,shutil,argparse,subprocess,re,collections
 import tempfile,hashlib,gzip
 
-sys.path.insert(0, os.path.dirname(os.path.abspath(__file__)))
```

Comment:
Maybe I'm missing it but what adds `cosalib` to the python path?  Are we now installing it as a global library?  Or is there some other place the path is inserted?

Reply:
Updating `/usr/lib/coreos-assembler/cmd-buildextend-vmware` to start with 

```python
#!/usr/bin/env python3
# NOTE: PYTHONUNBUFFERED is set in cmdlib.sh for unbuffered output
#
# An operation that mutates a build by generating an ova

import os
import sys
import json
import shutil
import argparse
import tarfile

from cosalib.cmdlib import run_verbose, load_json, write_json, sha256sum_file
from cosalib.builds import Builds

print(sys.path)
print(Builds)
raise SystemExit
```

Outputs the following:
```
[coreos-assembler]$ cosa buildextend-vmware
['/usr/lib/coreos-assembler', '/usr/lib64/python37.zip', '/usr/lib64/python3.7', '/usr/lib64/python3.7/lib-dynload', '/usr/lib64/python3.7/site-packages', '/usr/lib/python3.7/site-packages']
<class 'cosalib.builds.Builds'>
```

I believe it's being added as the "current path". Normally this shows up as `""`:
```
[coreos-assembler]$ python3
Python 3.7.3 (default, May 11 2019, 00:38:04) 
[GCC 9.1.1 20190503 (Red Hat 9.1.1-1)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import sys
>>> print(sys.path)
['', '/usr/lib64/python37.zip', '/usr/lib64/python3.7', '/usr/lib64/python3.7/lib-dynload', '/usr/lib64/python3.7/site-packages', '/usr/lib/python3.7/site-packages']
>>> 
```
However, I _think_ it's pulling in `/usr/lib/coreos-assembler` due to symlinking (as nothing sets it)
```
[coreos-assembler]$ ls -la /usr/bin/cosa             
lrwxrwxrwx. 1 root root 16 Jul 15 18:57 /usr/bin/cosa -> coreos-assembler
[coreos-assembler]$ ls -la /usr/bin/coreos-assembler 
lrwxrwxrwx. 1 root root 40 Jul 15 18:57 /usr/bin/coreos-assembler -> ../lib/coreos-assembler/coreos-assembler
```